modules/CurveAnalysis/src/client/CurveCuratorAppController.coffee

```diff
@@ -37,11 +37,11 @@ class CurveCuratorAppController extends Backbone.View
 	loadCurvesForExptCode: (exptCode, curveID) =>
 		UtilityFunctions::showProgressModal @$('.bv_loadCurvesModal')
 		@ccc.setupCurator(exptCode, curveID)
-		resultViewerURL = "/openExptInQueryTool?experiment="+exptCode
-		@$('.bv_resultViewerBtn').attr('href',resultViewerURL)
-		@$('.bv_resultViewerBtn').html('Open in '+window.conf.service.result.viewer.displayName)
-		@$('.bv_resultViewerBtn').show()
 
+		@openExperimentInQueryToolController = new OpenExperimentInQueryToolController
+			code: exptCode 
+			experimentStatus: 'approved'
```

Comment:
Why is `approved` hardcoded here?  It seems like `OpenExperimentInQueryToolController` should be able look up the experiment status if not provided, or we should pass the real experiment status here.

Reply:
@brianbolt this is valid feedback, but it looks like we have 'approved' hardcoded in several other places that use this controller. I'm inclined to merge this as-is, then spin off a separate techdebt case to address the hardcoded experiment statuses.

Reply:
I see. Makes sense. LGTM then.

Reply:
Resolving conversation to merge.